example/4-cascading-zipper/sfn/sfn_echo.go

```diff
@@ -8,12 +8,13 @@ import (
 )
 
 func main() {
-	sfn := yomo.NewStreamFunction("echo-sfn", yomo.WithZipperAddr("localhost:9002"))
+	sfn := yomo.NewStreamFunction(
+		"echo-sfn",
+		yomo.WithZipperAddr("localhost:9002"),
+		yomo.WithObserveDataTags(0x33),
```

Comment:
sfn should support multiple DataTags.

Reply:
parameters is slices, u can write `yomo.WithObserveDataTags(0x33,0x34,...)`